integration_test.go

```diff
@@ -158,15 +172,14 @@ func testHelperCartographer(
 		test.That(t, errors.New("test timeout"), test.ShouldBeNil)
```

Comment:
[minor] add comment here saying we will check both channels once a accurate mock dataset has been gathered and like the JIRA ticket for that

Reply:
Done